iothub/device/src/MethodResponseStatusCode.cs

```diff
@@ -8,8 +8,11 @@ namespace Microsoft.Azure.Devices.Client
     /// </summary>
     public enum MethodResponseStatusCode
     {
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
```

Comment:
Adding comments here seemed redundant. If someone has a better idea, let me know.

Reply:
The description for 500 = UserCodeException is a bit confusing to me; isn't 500 always a service error?

for the warning - how about we add it to the suppressions file instead, might look cleaner. It looks like we have one for device client: https://github.com/Azure/azure-iot-sdk-csharp/blob/master/iothub/device/src/GlobalSuppressions.cs

Reply:
What if we mention that these values are interchangable with actual http status codes?
Anyways, if you decide not to add comments for them, you can supress them in a single attribute on the class